### ISSUE TITLE
Typo in 'Basic JavaScript: Build JavaScript Objects'

### ISSUE THREAD

**Issue by joelszn** [freeCodeCamp/freeCodeCamp#38672]
Did you guys mean 'as long as "name" is a string' in last paragraph?
![Screen Shot 2020-04-29 at 1 32 25 AM](https://user-images.githubusercontent.com/23111335/80563578-671b0300-89b9-11ea-8932-e4d91646b8bd.png)

**Comment by thecodingaviator** [comment]
Yes, I think that's the intended motive. Please go ahead and create a PR or let us know so I'll put it up for other contributors!

**Comment by joelszn** [comment]
I'll create a PR :)

**Comment by vedanthv** [comment]
"as long as" conveys the message to the learner that name must be of the string datatype
